src/main/java/bryan/exception/EmptyDeadlineDescription.java

```diff
@@ -0,0 +1,4 @@
+package bryan.exception;
+
+public class EmptyDeadlineDescription extends Exception{
+}
```

Comment:
pertains to all exception classes. Consider at least having a message that distinguishes the exceptions